user-operator/src/main/java/io/strimzi/operator/user/UserOperatorConfig.java

```diff
@@ -21,7 +21,7 @@
     public static final String STRIMZI_CA_CERT_SECRET_NAME = "STRIMZI_CA_CERT_NAME";
     public static final String STRIMZI_CA_KEY_SECRET_NAME = "STRIMZI_CA_KEY_NAME";
     public static final String STRIMZI_CLUSTER_CA_CERT_SECRET_NAME = "STRIMZI_CLUSTER_CA_CERT_SECRET_NAME";
-    public static final String STRIMZI_EO_KEY_SECRET_NAME = "STRIMZI_EO_KEY_SECRET_NAME";
+    public static final String STRIMZI_EUO_KEY_SECRET_NAME = "STRIMZI_EUO_KEY_SECRET_NAME";
```

Comment:
There are users using the User Operator _standalone_ with Kafka clusters not deployed by Strimzi. They use the files for it from here: https://github.com/strimzi/strimzi-kafka-operator/tree/main/packaging/install/user-operator. Changing the environment variable name is basically a backwards-incompatible change of API for them. It is also covered in the docs https://strimzi.io/docs/operators/latest/full/deploying.html#deploying-the-user-operator-standalone-str.

I think the best thing would be to keep the old environment variable as it was. That way, there is no need to change the docs and no breaking-change for the standalone user operator users.

Reply:
Sounds good! I will bring it back. Basically, I will do this in the UserOperatorConfig.java
```
        String euoKeySecretName = map.get(UserOperatorConfig.STRIMZI_EUO_KEY_SECRET_NAME);
        if (euoKeySecretName == null || euoKeySecretName.isEmpty()) {
            // It's needed for the backward compatible with the standAlone deployment of the User Operator.
            euoKeySecretName = map.get(UserOperatorConfig.STRIMZI_EO_KEY_SECRET_NAME);
        }
```

I guess this should work in both modes: StandAlone mode vs Strimzi mode.

Reply:
In general, yes ... but wouldn't it be just easier to use the original name?

Reply:
ah, yes! Let me change it.

Reply:
done.